lib/plugins.ts

```diff
@@ -508,7 +509,10 @@ export class Plugins {
               .map(
                   pluginObj =>
                       this.safeCallPluginFun(pluginObj, funName, args, promiseType, failReturnVal));
-      return promiseType == PromiseType.Q ? q.all(promises) : webdriver.promise.all(promises);
+      return promiseType == PromiseType.Q ?
```

Comment:
It can, but the promise that Promise.all will return is an ES6 promise. If there are no expectations in the future, about retrieving a promise which has support for the large array of Q methods, that sounds like a bigger conversation than an internal tweak. That said, I have no qualms with stripping the vestiges of Q from anything that might be externally-facing, if so desired.